src/tree/containers/ContainerProperties.ts

```diff
@@ -20,6 +20,7 @@ export const containerProperties: ITreePropertyInfo<ContainerProperty>[] = [
     { property: 'State', exampleValue: 'exited' },
     { property: 'Status', exampleValue: 'Exited (0) 2 hours ago' },
     { property: 'Compose Project Name', description: l10n.t('Value used to associate containers launched by a \'docker-compose up\' command') },
+    { property: 'Label', exampleValue: 'com.microsoft.created-by' },
```

Comment:
I'd include the label value so it's clear that we're grouping by label _and_ value
```suggestion
    { property: 'Label', exampleValue: 'com.microsoft.created-by=visual-studio-code' },
```

Reply:
Wouldn't this encourage users to put `com.microsoft.created-by=visual-studio-code` instead of `com.microsoft.created-by` in the settings? which will cause the Group by Label feature to not work?

Reply:
That's fair. The others are an example end value though. Maybe we could put the example setting value in the description for the setting?

Reply:
That's a great idea actually. I'll do that